### DISCUSSION TITLE
Problem with text rendering

### DISCUSSION THREAD

**Discussion by eli-yip** [sharkdp/bat#2534] [Q&A]
I have used `bat` for some time on my Mac and it works well. Then today I try to install it on my brand new ubuntu server, which version is 22.04 LTS and have nothing conflicted.

When I try to batcat something such as `.zshrc` file, it happens to me that all the black text are rendered as white, while anything else seems to work fine.

I have searched the Internet and changed my default terminal theme to white, which means text color is black and background color is white. But it doesn't help?

What should I do? I am a fresh to ubuntu and not a native english speaker. If any more information is needed, I am happy to provide.

Thank you!

**Comment by keith-hall** [comment]
Hi, sorry for the delay in replying. Have you tried configuring `bat` to use a different theme to see if that helps?
